src/OpenTelemetry.Api/Trace/TracerProvider.cs

```diff
@@ -35,6 +35,20 @@ protected TracerProvider()
         /// </summary>
         public static TracerProvider Default { get; } = new TracerProvider();
 
+        internal static int? SpanAttributeCountLimit { get; set; }
+
+        internal static int? SpanAttributeValueLengthLimit { get; set; }
+
+        /// <summary>
+        /// Configures the <see cref="SpanLimits" /> for the default <see cref="TracerProvider" />.
+        /// </summary>
+        /// <param name="spanLimits">The configured <see cref="SpanLimits" />.</param>
+        public static void SetSpanLimits(SpanLimits spanLimits)
+        {
+            SpanAttributeCountLimit = spanLimits.AttributeCountLimit;
```

Comment:
How to handle error when the value is out of range?

Reply:
Good question. My thought was that if set to negative then it it won't hurt anything - i.e., the setting would have no effect and nothing would be limited. At least that was my intent... though just realized that setting `SpanAttributeValueLengthLimit` to a negative value will cause an exception upon calling `String.Substring`. Will fix.

I'm open to other opinions, but since this is part of the API and not the SDK, I figured an error here was not appropriate.